packages/ember-views/lib/views/view.js

```diff
@@ -1214,48 +1215,36 @@ var View = CoreView.extend({
     @private
   */
   _applyAttributeBindings: function(buffer, attributeBindings) {
-    var attributeValue;
     var unspecifiedAttributeBindings = this._unspecifiedAttributeBindings = this._unspecifiedAttributeBindings || {};
 
-    forEach(attributeBindings, function(binding) {
-      var split = binding.split(':');
-      var property = split[0];
-      var attributeName = split[1] || property;
+    var binding, split, property, attrName, attrNode, attrValue, value;
+    var i, l;
+    for (i=0, l=attributeBindings.length; i<l; i++) {
+      binding = attributeBindings[i];
+      split = binding.split(':');
+      property = split[0];
+      attrName = split[1] || property;
```

Comment:
Good thinking!  Since `:` isn't valid a valid character for property names, they'd have to use a view-prop anyway, right?  In that case the gotcha's not much of a limitation, since there's no way they could define the property for `xlink:href` under its own name even if they wanted to.  (If I remember the whole mapping thing right, that is)